samples/Azure.Management.Storage/Generated/BlobContainerContainer.cs

```diff
@@ -179,6 +179,8 @@ public virtual Response<BlobContainer> Get(string containerName, CancellationTok
                 }
 
                 var response = _restClient.Get(Id.ResourceGroupName, Id.Name, containerName, cancellationToken: cancellationToken);
+                if (response.Value == null)
+                    throw _clientDiagnostics.CreateRequestFailedException(response.GetRawResponse());
```

Comment:
What does it mean when service returns null value? is it 404 resource not found? if yes, then should we throw something like resource not found exception, instead of request failed exception?